misc/ref/example_seismic.py.out

```diff
@@ -0,0 +1,3 @@
+pressure is not defined for IASP91
+gravity is not defined for IASP91
+density is not defined for IASP91
```

Comment:
Yes, I put in warnings whenever it computes these values and they aren't
given. This is to warn if you put in your own, say upper mantle model, and
these calculations go totally wrong. I guess I could put in tests if it is
a whole earth model, but then again, we should maybe also add a moon model.

On Tue, Sep 15, 2015 at 7:54 PM, Ian Rose <notifications@github.com> wrote:

> In misc/ref/example_seismic.py.out
> <https://github.com/geodynamics/burnman/pull/107#discussion_r39550172>:
>
> > @@ -0,0 +1,3 @@
> > +pressure is not defined for IASP91
> > +gravity is not defined for IASP91
> > +density is not defined for IASP91
>
> When I run the script I also get warning output about pressure and gravity
> for STW105 and AK135
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/burnman/pull/107/files#r39550172>.
>